test/freecoin/test_helpers/integration.clj

```diff
@@ -36,7 +37,7 @@
 (defn default-app-config-m []
   (let [stores (s/create-in-memory-stores)]
     {:stores-m stores
-     :blockchain (blockchain/create-in-memory-blockchain :bk)
+     :blockchain (InMemoryBlockchain. :bk (atom {}) (atom {}) (atom {}))
      :config-m {:secure "false"
```

Comment:
yep! thanks for spotting this, was leftover as a test about the problem you then found how to solve...